File: poo2/projeto/php_netbeans/modelo/Polia.php
<?php
require_once 'Produto.php';
require_once 'Fabricante.php';
require_once 'interface/polia_interface.php';
class Polia extends Produto implements polia_interface{
     public $qte_sulcos;
        function Polia($numero_peca, $altura, $largura, $comprimento, $diamentroInterno, $diamentroExterno,$amperagem,$voltagem,$valor,$marca,$numero_oem,$qte_sulcos){
            $this->numero_peca      = $numero_peca;
            $this->altura           = $altura;
            $this->largura          = $largura;
            $this->comprimento      = $comprimento;
            $this->diamentroInterno = $diamentroInterno;
            $this->diamentroExterno = $diamentroExterno;
            $this->amperagem        = $amperagem;
            $this->voltagem         = $voltagem;
            $this->valor            = $valor;
            $this->qte_sulcos       = $qte_sulcos;             
            $this->fabricante       = new Fabricante($marca, $numero_oem);
        } 
}
?>

File: projeto/ServeletAutopecas/src/java/LN/Polia.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package LN;

/**
 *
 * @author marcelo
 */
public class Polia extends Produto {
    int qte_sulcos;

    public Polia(int numeroPeca, String altura, String largura, String comprimento,
                          String diamentroInterno, String diamentroExterno, float valor, float amperagem,
                          float voltagem, int qte_sulcos,String nome,  String numeroOEM) {
        super.setNumeroPeca(numeroPeca);
        super.setAltura(altura);
        super.setLargura(largura);
        super.setComprimento(comprimento);
        super.setDiamentroExterno(diamentroExterno);
        super.setDiamentroInterno(diamentroInterno);
        super.setValor(valor);
        super.setAmperagem(amperagem);
        super.setVoltagem(voltagem);
        setQte_sulcos(qte_sulcos);
    }

    public int getQte_sulcos() {
        return qte_sulcos;
    }

    public void setQte_sulcos(int qte_sulcos) {
        this.qte_sulcos = qte_sulcos;
    }

  
    
}
